lib/modules/platform/gitlab/index.ts

```diff
@@ -538,9 +538,20 @@ async function ignoreApprovals(pr: number): Promise<void> {
       }[]
     >(url);
 
+    const ruleName = 'renovateIgnoreApprovals';
+
     const existingAnyApproverRule = rules?.find(
       ({ rule_type }) => rule_type === 'any_approver'
     );
+    const existingRegularApproverRules = rules?.filter(
+      ({ rule_type, name }) => rule_type !== 'any_approver' && name !== ruleName
+    );
+
+    await Promise.all(
+      existingRegularApproverRules?.map(async (rule) => {
```

Comment:
`Promise.all(undefined)` rejects, so add a condition to only do `if existingRegularApproverRules?.length`